src/components/Settings/SettingsPanel/SettingsPlayersSections/SettingsPlayersSections.tsx

```diff
@@ -0,0 +1,132 @@
+
+  return (
+    <React.Fragment>
+      <ReactSortable
+        handle=".list-handle"
+        delayOnTouchOnly={isMobile}
+        delay={isMobile ? 10 : 0}
+        forceFallback={true}
+        list={playersLists}
+        setList={(newState) => dispatch(setPlayersLists(newState))}
+        className={classes.SettingsPlayersLists}
+      >
+        {playersLists.map((list) => (
+          <div key={list.id} className={classes.SettingsPlayersListItem}>
+            <div
+              className={`list-handle ${classes.SettingsPlayersListsHandle}`}
+            >
+              <FontAwesomeIcon icon="sort" size="lg" />
+            </div>
+
+            <div className={classes.SettingsInputContainer}>
+              <input
+                type="text"
+                placeholder="Section name"
+                className={classes.SettingsInput}
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                  dispatch(
+                    setPlayersListsTitle({
+                      index: list.id as number,
+                      title: event.currentTarget.value,
+                    })
+                  );
+                }}
+                value={t(list.title) as string}
+              />
+            </div>
+
+            <div
+              className={classes.SettingsPlayersContainer}
+              data-selected={playersContainer === list.id}
+              onClick={() => dispatch(setPlayersContainer(list.id as number))}
+            >
+              <FontAwesomeIcon icon="users" />
+            </div>
+
+            <div>
+              <Button
+                danger
+                className={classes.SettingsDeleteButton}
+                onClick={() => {
+                  if (list.id !== playersContainer) {
+                    dispatch(
+                      setPlayersLists([
+                        ...playersLists.filter(
+                          (element) => element.id !== list.id
+                        ),
+                      ])
+                    );
+                  }
+                }}
+              >
+                <FontAwesomeIcon icon="times" size="sm" />
+              </Button>
+            </div>
+          </div>
+        ))}
+      </ReactSortable>
+
+      <div className={classes.SettingsSectionsButtons}>
+        <Button
+          onClick={() => {
+            dispatch(
+              setPlayersLists([
+                ...playersLists,
+
+                { id: playersLists.length, title: "", players: [] },
+              ])
+            );
+          }}
+        >
+          {t("settings.addSection")}
+        </Button>
+        <Button
+          danger
+          onClick={() => {
+            dispatch(resetPlayersLists());
+          }}
+        >
+          {t("settings.resetSections")}
+        </Button>
+      </div>
+
+      <p>
+        <small>
+          <Trans i18nKey="settings.sectionsNote">
+            Note: changing the players container (
```

Comment:
I found the wording confusing on the first read.

Would it be better to change players container to something else, such as default players container/default container?

Reply:
so just add default?

Reply:
that or rephrase it so it's clearer? On first impression, it didn't read clear enough what that icon indicates

Reply:
What about:

`Note: changing the default section (icon) will only be affected after clicking Reset Round or Reset All.`